mkdocs.yml

```diff
@@ -107,6 +107,12 @@ nav:
             - 'Logo customization': variants/clevo_nv41/custom_logo.md
             - 'Openness score': variants/clevo_nv41/openness_analysis.md
             - 'LVFS report': variants/clevo_nv41/lvfs_report.md
+        - 'Tuxedo IBS15':
```

Comment:
@mkopec I would like to understand why Tuxedo deserve section but Novacustom NV41 is under Clevo NV41. Please be consistent in documentation when you apply some rule, best way to enforce that on others is to add guideline how to approach creation of new section. Last resort all that stuff should be scripted.

Reply:
Yes, please rename code branches - in light of having potentially multiple OEMs based on Clevo we should improve structure of code to handle that case. @macpijan cc

I know that Felix S. prepared code structure in coreboot for above case, but really don't know it works in practice.

Reply:
> I would like to understand why Tuxedo deserve section but Novacustom NV41 is under Clevo NV41. Please be consistent in documentation when you apply some rule

This is evolving process. With each new machine and project we see more cases to cover. For the NS50 we can see we should use NovaCustom / Tuxedo, and we should probably go back and to the same for NV40. But that would mean we also need to make `novacustom/nv41` mainboard based on the `clevo/nv41`. Wchich can be considered redundant by some, if there is only one such board supported.

> Yes, please rename code branches - in light of having potentially multiple OEMs based on Clevo we should improve structure of code to handle that case.
> I know that Felix S. prepared code structure in coreboot for above case, but really don't know it works in practice.

We already  do that for the NS50 release. We may also change the mainbaord and brancing for the past NV41 but it needs to me done carefuly so we do not break anything.

Reply:
Understood, just please optimize layout whenever you can to avoid duplication and incorrect labeling. Please note this repo is part of our customers marketing.